lib/segment/src/data_types/tiny_map.rs

```diff
@@ -0,0 +1,146 @@
+use tinyvec::TinyVec;
+
+const CAPACITY: usize = 6;
```

Comment:
I propose to decrease to 3 because most user cases are between 1 and 3. We reduce stack usage